bibs/Idstein.json

```diff
@@ -10,7 +10,17 @@
     "city": "Idstein",
     "country": "Deutschland",
     "data": {
-        "baseurl": "http://stadtbuecherei.idstein.de",
+        "baseurl": "https://stadtbuecherei.idstein.de",
+        "mediatypes": {
+            "0": "ALL",
+            "1": "BOOK",
+            "11": "DVD",
+            "19": "BOOK",
+            "2": "MAGAZINE",
+            "4": "MOVIE",
+            "5": "AUDIO_CASSETTE",
+            "6": "CD"
+        },
```

Comment:
mediatypes look wrong (see longer comment above), at least `ALL` needs to be removed.

Reply:
See my [comment](https://github.com/opacapp/opacapp-config-files/pull/11#discussion_r430272819) at Essen.

Reply:
see reply above.